### PR TITLE
Fix macOS cmd + <key> not sending <key> pressed

### LINKED ISSUE #78: Key combinations with `cmd` not fired correctly on macOS
For example, `Cmd + C` (`Cmd` as `KEY_LMETA` or `KEY_RMETA`) sends only three events:

- KEY_LMETA pressed
- KEY_C released
- KEY_LMETA released

### PR DESCRIPTION
**Pull request by ghostflyby**
closes #78

### CODE REVIEW

File: src/main/java/org/lwjglx/opengl/Display.java [hunk 1]
```diff
@@ -169,7 +169,9 @@ public void invoke(long window, int key, int scancode, int action, int mods) {
                 cancelNextChar = false;
                 if (key > GLFW_KEY_SPACE && key <= GLFW_KEY_GRAVE_ACCENT) { // Handle keys have a char. Exclude space to
                                                                             // avoid extra input when switching IME
-                    if ((GLFW_MOD_CONTROL & mods) != 0 && (GLFW_MOD_ALT & mods) == 0) { // Handle ctrl + x/c/v.
+                    if ((GLFW_MOD_SUPER & mods) != 0) {
+                        Keyboard.addGlfwKeyEvent(window, key, scancode, action, mods, (char) key);
```
Comment (eigenraven) [inline src/main/java/org/lwjglx/opengl/Display.java:173]: `cancelNextChar = true'` is probably also needed here because you already specify a char code.

Comment (ghostflyby) [reply src/main/java/org/lwjglx/opengl/Display.java:173]: At least on macOS, the Cmd+<key> never sends the char event (before or after the fix), which is the cause of the bug.

Comment (eigenraven) [reply src/main/java/org/lwjglx/opengl/Display.java:173]: I will have to test this on other platforms to make sure this code doesn't break windows/linux

Comment (ghostflyby) [reply src/main/java/org/lwjglx/opengl/Display.java:173]: Probably just linux, as `GLFW_MOD_SUPER` is mapped to META, aka Cmd on macOS and Win key on Windows, which can hardly be in combination with other keys.

**Review by eigenraven — APPROVED** [review]

**Pull request merged**